nelc/frontend-essentials
# feat: Add webpack and bundle file

## Description
This is the basic implementation of this decision  https://github.com/nelc/frontend-essentials/pull/5

## How to test

1.  Check this branch
2. Run `npm run build-bundle`

The previous command should create a new file within the folder dist called  frontend-essentials.js, that will contain the exported components inside bundle.tsx
